src/main.rs

```diff
@@ -1,22 +1,23 @@
+#[macro_use]
+extern crate dotenv_codegen;
+
 use iroha::*;
 use std::thread;
 use std::time::Duration;
 
 fn main() {
     println!("Hyperledgerいろは2にようこそ！");
-
     // Setup data structures
-    let block_time_ms = 1000; //TODO: read from config file
-
+    let block_time_ms = dotenv!("BLOCK_BUILD_STEP_MS")
```

Comment:
Just read in and parse the config.json and trusted_peers.json here at the beginning of main. Then these variables can be used to initialize the iroha node.